system/lib/dlmalloc.c

```diff
@@ -2441,7 +2441,7 @@ typedef unsigned int flag_t;           /* The type of various bit flag sets */
  of the same size are arranged in a circularly-linked list, with only
  the oldest chunk (the next to be used, in our FIFO ordering)
  actually in the tree.  (Tree members are distinguished by a non-null
- parent pointer.)  If a chunk with the same size an an existing node
```

Comment:
Since this is a copy of upstream code I'm tempted to just leave this unmodified.  Can you revert this line?